cmd/state-svc/test/integration/svc_int_test.go

```diff
@@ -207,40 +208,61 @@ func (suite *SvcIntegrationTestSuite) TestAutostartConfigEnableDisable() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
+	autostartDir := filepath.Join(ts.Dirs.Config, "autostart")
+	err := fileutils.Mkdir(autostartDir)
+	suite.Require().NoError(err)
+	err = os.Setenv("_TEST_AUTOSTART_DIR", autostartDir)
```

Comment:
Please add it to constants.go and follow our naming convention (ie. prefix with ACTIVESTATE).